src/options/OptionHelper.js

```diff
@@ -71,7 +71,9 @@ class OptionHelper {
 
     // Operate on a copy of the plugin, since the webpack task
     // can be called multiple times for one instance of a plugin
-    const instance = Object.create(plugin);
+    const instance = Array.isArray(plugin) ? [] :
+      Object.prototype.isPrototypeOf(plugin) ? Object.create(plugin) :
+      plugin;
```

Comment:
This is already resolved, but just for my own knowledge, why wouldn't this work? `instance` would be created as an empty array, and then the `instance` would be populated in line 77 with `Object.keys(plugin).forEach(key => ...)`